Source/NSBitmapImageRep+BMP.m

```diff
@@ -0,0 +1,444 @@
+/*
+   NSBitmapImageRep+PNG.m
```

Comment:
Wrong name

Reply:
Oh... that wasn't done by me, but I'll correct it.  It looks like a mistake that might have been made a long time ago.

Reply:
File removed.